client/src/components/selects/codeSystemSelect.tsx

```diff
@@ -30,7 +30,8 @@ interface Props {
 }
 
 const filterByName: ItemPredicate<CodeSystem> = (query, item) => {
-  return `${item.title.toLowerCase()}`.indexOf(query.toLowerCase()) >= 0;
+  const title = item.title || item.name;
```

Comment:
Are we sure that there will always be either a title or a name? They aren't required attributes

Reply:
At least it works with the code systems we have now but it's indeed not a very robust solution...
I'll try to think about a better way to handle this

Reply:
add //TODO comment